srcpkgs/zrepl/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'zrepl'
+pkgname=zrepl
+version=0.1.1
+revision=1
+archs="x86_64* aarch64*"
```

Comment:
The archs field is for packages that are inherently bound to specific platforms (an x86 specific bootloader or a glibc compatibility library on musl platforms for example) This package seems just to be broken for 64 bit platforms.

So replace the archs flag by:
```bash
case "$XBPS_TARGET_MACHINE" in
	x86_64*|aarch64*) broken="XXX" ;;
esac
```
Where XXX is a short description of the build failure or ideally a link to the failing build log.